library/testing/TestSDKCamera.cxx

```diff
@@ -179,8 +179,30 @@ int TestSDKCamera(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
+  // Test getAzimuth
+  f3d::angle_deg_t addAzimuth = cam.getAzimuth();
+  if (!compareDouble(addAzimuth, 0.0))
+  {
+    std::cerr << "getAzimuth is not behaving as expected:" << std::endl;
+    std::cerr << std::setprecision(12) << "azimuth: " << addAzimuth << " != 0.0" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Test getYaw
+  f3d::angle_deg_t addYaw = cam.getYaw();
+  if (!compareDouble(addYaw, 90.0))
+  {
+    std::cerr << "getYaw is not behaving as expected:" << std::endl;
+    std::cerr << std::setprecision(12) << "yaw: " << addYaw << " != 0.0" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Test getElevation
+  double addElevation = cam.getElevation();
+  checkDouble(addElevation, 0.0, "getElevation");
```

Comment:
please put this test after the test of addElevation

Reply:
I added it after addElevation.